src/components/widgetCRUD/services/widgetConfService.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Cognite AS
+import { Layout } from 'react-grid-layout';
+import findIndex from 'lodash/findIndex';
+import { WidgetConfig } from 'components/grid/interfaces';
+import { getByUser, update, deleteOne, save } from '../repo/widgstConfFSRepo';
```

Comment:
Isn't there a better way to inject the repo as an interface so that the service does not have to hardcode the actual implementation it uses?

Reply:
These are not class-based modules. this is the simplest way I've found. we use local storage only for developing purposes. So I thought this is enough for now because there is only one position we want to change.

Reply:
is using class based modules discouraged? Even in that case there should be good ways to implement this type of functionality.

Reply:
Unfortunately, I couldn't find a good way. However, I can implement the class-based approach?

Reply:
Lets create a task to research/work on abstracting out config saving mechanism and close this PR.